test/extended/util/framework.go

```diff
@@ -926,7 +926,7 @@ func WaitForABuild(c buildv1clienttyped.BuildInterface, name string, isOK, isFai
 		return err
 	}
 	// wait longer for the build to run to completion
-	err = wait.Poll(5*time.Second, 10*time.Minute, func() (bool, error) {
+	err = wait.Poll(5*time.Second, 15*time.Minute, func() (bool, error) {
```

Comment:
fyi @alicerum I think it is worthwhile to capture a build log where we see buildah's attempt to pull the image from `quay.io/openshift/community-e2e-test` and show it to @nalind to see if he thinks the output produced when it is pulling the image, and that it takes so long, is indicative of an issue that needs to be looked at on the buildah side (vs. quay.io just being slow).

You could run the test manually and use `--include-success` along with perhaps some short term changes to the test so it prints the build log even when it succeeds.

@adambkaplan FYI